video_xblock/static/css/brightcove.css

```diff
@@ -493,3 +493,7 @@ body .video-js .vjs-menu-button-popup .vjs-menu .vjs-menu-item:hover {
 .display-flex {
   display: flex;
 }
+
+.vjs-text-track-display div div {
```

Comment:
I don't really like such kind of selectors. But fix requires tweaking videojs' transcripts, which is too much at the moment.